src/app/zap-templates/zcl/data-model/chip/multicast-cluster.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
Drive by review - I realize this is still in draft format, however quick notes which could maybe be addressed now:

- PR for declaring XML should be separated out from the implementation
- XMLs should be generated via alchemy, not manual. Alchemy tag should exist here
- Clusters should be unit tested. This is super hard on existing pure-codegen, however #39163 starts adding documentation on designing clusters that are easier to test. Consider if that approach is possible here (not yet mandated, however I imagine we will soon as we really want to start having unit testable things ... we are just waiting for full attribute persistence to be solved). An example of tested cluster is https://github.com/project-chip/connectedhomeip/tree/master/src/app/clusters/software-diagnostics-server